packages/share/lib/share.dart

```diff
@@ -51,4 +52,79 @@ class Share {
 
     return channel.invokeMethod<void>('share', params);
   }
+
+  /// Summons the platform's share sheet to share a single file.
+  ///
+  /// Wraps the platform's native share dialog. Can share a file.
+  /// It uses the `ACTION_SEND` Intent on Android and `UIActivityViewController`
+  /// on iOS.
+  ///
+  /// The optional `sharePositionOrigin` parameter can be used to specify a global
+  /// origin rect for the share sheet to popover from on iPads. It has no effect
+  /// on non-iPads.
+  ///
+  /// May throw [PlatformException] or [FormatException]
+  /// from [MethodChannel].
+  static Future<void> shareFile(
```

Comment:
I'm not sure the `shareFile` provides a lot of value. Why not remove it in favor of only one single `shareFiles` method that accepts an array?

(Also look at the example, it makes the code a little bit more complex having to decide whether to call `shareFile` vs `shareFiles`

Reply:
I think `shareFile()` was there first, and `shareFiles()` was contributed later. That may explain its existence. I do agree that `shareFiles()` works with an array of one file, too :+1:

Reply:
Yes I think `shareFile()` was there first. I left it for making things easier when just sharing one file. But I agree that it just removes `[]` which is not enough to keep it.